analyze/analyze.dart

```diff
@@ -256,6 +290,13 @@ Future<void> _checkForNewExecutables() async {
       'must add this file to kExecutableAllowlist in analyze/analyze.dart',
     );
   }
+  if (unexpectedShebangShellCount > 0) {
+    throw Exception(
+      'found $unexpectedShebangShellCount unexpected shell #! line'
+      '${unexpectedShebangShellCount == 1 ? '' : 's'}! If this was intended, you '
+      'must modify kShebangRegex in analyze/analyze.dart',
+    );
+  }
```

Comment:
This dups the above lines.

Reply:
No, it doesn't.  It's a different message.

Reply:
Oops, my bad.